site/build.gradle

```diff
@@ -67,6 +67,13 @@ task javadoc(type: Javadoc,
         addBooleanOption('Xdoclint:all').value = true
         addBooleanOption('Xdoclint:-missing').value = true
 
+        //add support for java8u121 "New --allow-script-in-comments option for javadoc"
+        //ref: http://www.oracle.com/technetwork/java/javase/8u121-relnotes-3315208.html
+        if (javax.tools.ToolProvider.getSystemDocumentationTool()
+                .isSupportedOption("--allow-script-in-comments")) {
+            addBooleanOption('-allow-script-in-comments').value = true
```

Comment:
I add this because it would fail when building Armeria under JDK8u121 environment without this. I 've recently updated my Oracle JDK to 8u121 and found this issue.